tests/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "4.1.0",
   "description": "simple, flexible, fun brightscript test framework for roku scenegraph apps - tests",
   "scripts": {
-    "build": "bsc"
+    "build": "bsc",
+    "watch": "npx bsc --project bsconfig.json --watch"
```

Comment:
Do you need the `--project bsconfig.json`? That's the default path for it anyway.

Reply:
oh - I guess not.